provider/provider_yaml_test.go

```diff
@@ -35,17 +37,65 @@ func Test_monitoring(t *testing.T) {
 }
 
 func Test_network(t *testing.T) {
-	test(t, filepath.Join("test-programs", "network"))
+	test(t, filepath.Join("test-programs", "network"),
+		optproviderupgrade.NewSourcePath(filepath.Join("test-programs", "network", "v6")))
 }
 
 func Test_servicebus(t *testing.T) {
 	test(t, filepath.Join("test-programs", "servicebus"))
 }
 
 func Test_sql(t *testing.T) {
-	test(t, filepath.Join("test-programs", "sql"))
+	test(t, filepath.Join("test-programs", "sql"),
+		optproviderupgrade.NewSourcePath(filepath.Join("test-programs", "sql", "v6")))
+}
+
+func Test_sql_managed(t *testing.T) {
+	t.Skip("Waiting for more quota, Azure support request 2409140020000074")
+	test(t, filepath.Join("test-programs", "sql_managed"),
+		optproviderupgrade.NewSourcePath(filepath.Join("test-programs", "sql_managed", "v6")),
+	)
 }
 
 func Test_storage(t *testing.T) {
-	test(t, filepath.Join("test-programs", "storage"))
+	test(t, filepath.Join("test-programs", "storage"),
+		optproviderupgrade.NewSourcePath(filepath.Join("test-programs", "storage", "v6")))
+}
+
+func Test_automation_runbook(t *testing.T) {
+	test(t, filepath.Join("test-programs", "automation_runbook"))
+}
+
+func Test_cdn_profile(t *testing.T) {
+	test(t, filepath.Join("test-programs", "cdn_profile"),
+		optproviderupgrade.NewSourcePath(filepath.Join("test-programs", "cdn_profile", "v6")))
+}
+
+func Test_compute_datadiskattachment(t *testing.T) {
+	test(t, filepath.Join("test-programs", "compute_datadiskattachment"),
+		optproviderupgrade.NewSourcePath(filepath.Join("test-programs", "compute_datadiskattachment", "v6")))
+}
+
+func Test_loadbalancer(t *testing.T) {
+	test(t, filepath.Join("test-programs", "loadbalancer"),
+		optproviderupgrade.NewSourcePath(filepath.Join("test-programs", "loadbalancer", "v6")))
+}
+
+func Test_iot_iothub(t *testing.T) {
+	test(t, filepath.Join("test-programs", "iot_iothub"))
+}
+
+func Test_cosmosdb(t *testing.T) {
+	t.Skip("Can't record as all tested regions are returning 'ServiceUnavailable' error")
```

Comment:
Should this be linked to an issue?

Reply:
Yeah, probably. It might just have been a bad day for Azure as it was specifically saying it was unable to allocate resources.